applets/brightness/brightness-applet.c

```diff
@@ -431,11 +454,7 @@ gpm_applet_plus_cb (GtkWidget *w, GpmBrightnessApplet *applet)
 static gboolean
 gpm_applet_minus_cb (GtkWidget *w, GpmBrightnessApplet *applet)
 {
-	if (applet->level > 0) {
-		applet->level--;
-	}
-	applet->call_worked = gpm_applet_set_brightness (applet);
-	gpm_applet_update_popup_level (applet);
+	gpm_applet_adjust_brightness(applet, -GPM_BRIGHTNESS_APPLET_PERCENT_CHANGE_BUTTON);
 	return TRUE;
```

Comment:
Same here.

Reply:
Hmm, and why there is a minus before GPM_.....?

Reply:
Or stands this for a negative value?

Reply:
Yep, it is just the negative direction here. 

I've just rebased and added in the two missing spaces.